### PR TITLE
Signal handling

### PR DESCRIPTION
**Pull request by danReynolds**
Handles CTRL+C and CTRL+Z, doing nothing. They still work for processes run within the shell.

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -1,10 +1,14 @@
 #![allow(unused_features)]
+#![feature(std_misc)]
 #![feature(static_mutex)]
 #![feature(box_syntax)]
+#![feature(libc)]
```
Comment (nikklassen) [inline src/main.rs]: Aren't we supposed to use libc from crates.io?

**Comment by nikklassen** [comment]
This looks fine.  It may be beneficial to at least print which signal was handled instead of doing a complete no-op

File: src/main.rs (hunk 1 above)
Comment (danReynolds) [inline src/main.rs]: This just allows libc to be used I believe. I'm using extern crate libc.

**Comment by danReynolds** [comment]
Okay, still a WIP until I can get some process management going.

File: src/main.rs (hunk 1 above)
Comment (nikklassen) [inline src/main.rs]: But you did't include libc in Cargo.toml.  You need that to be using the crates.io one.

Comment (danReynolds) [inline src/main.rs]: which one am I using currently then? The system one? What's wrong with that one?

Comment (nikklassen) [inline src/main.rs]: Ya, you're using rust's builtin libc.  I'm just going off of the warning on http://doc.rust-lang.org/nightly/libc/index.html.

**Comment by nikklassen** [comment]
We need to update nix-rust once the version gets updated, btw.  They merged in my PR to fix the signal handling on Mac.

**Comment by nikklassen** [comment]
Also need to catch SIGPIPE and exit cleanly

**Pull request merged**